modules/claims_api/app/sidekiq/claims_api/report_hourly_unsuccessful_submissions.rb

```diff
@@ -68,32 +67,32 @@ def allow_processing?
     end
 
     def va_gov_errored_claims
-      errored_claims = get_unique_errors || []
-      @va_gov_errored_claims = [] if @va_gov_errored_claims.blank?
-      @va_gov_errored_claims << errored_claims
-      if @va_gov_errored_claims.flatten.blank?
-        {}
-      else
-        @va_gov_errored_claims.flatten&.group_by(&:transaction_id)
-        @va_gov_errored_claims&.map { |grp| grp[1][0] }&.pluck(:id)
-      end
+      errored_claims = get_unique_errors
+      errored_claims.map { |record| record[0] } # rubocop:disable Rails/Pluck
```

Comment:
Why not  `get_unique_errors.pluck(:id)`? Or better yet, return `errored_claims.pluck(:id)` from `get_unique_errors` and remove this method? 🤷